src/python/pants_release/release.py

```diff
@@ -809,18 +818,59 @@ def prompt_to_generate_docs() -> None:
 
 def test_release() -> None:
     banner("Installing and testing the latest released packages")
-    install_and_test_packages(CONSTANTS.pants_stable_version)
-    banner("Successfully installed and tested the latest released packages")
+    smoke_test_install_and_version(CONSTANTS.pants_stable_version)
+    banner("Successfully ran a smoke test of the released packages")
 
 
-def install_and_test_packages(version: str, *, extra_pip_args: list[str] | None = None) -> None:
-    with create_tmp_venv() as bin_dir:
-        for pkg in PACKAGES:
-            pip_req = f"{pkg.name}=={version}"
-            banner(f"Installing and testing {pip_req}")
-            pkg.validate(version, bin_dir, extra_pip_args or [])
-            green(f"Tests succeeded for {pip_req}")
+def smoke_test_install_and_version(version: str) -> None:
+    """
+    Do two tests to confirm that both sets of artifacts (PEXes for running normally, and wheels
+    for plugins) have ended up somewhere plausible, to catch major infra failures
+    """
+    with temporary_dir() as dir_:
+        dir = Path(dir_)
+        (dir / "pants.toml").write_text(
+            f"""
+            [GLOBAL]
+            pants_version = "{version}"
+
+            backend_packages = [
+                "pants.backend.python",
+                "pants.backend.plugin_development",
+            ]
 
+            [python]
+            interpreter_constraints = ["==3.9.*"]
+            """
+        )
+
+        # First: test that running pants normally reports the expected version:
+        result = subprocess.run(
+            ["pants", "version"],
+            cwd=dir,
+            check=True,
+            stdout=subprocess.PIPE,
+        )
+        printed_version = result.stdout.decode().strip()
+        if printed_version != version:
+            die(f"Failed to confirm pants version, expected {version!r}, got {printed_version!r}")
+
+        # Second: test that the wheels can be installed/imported (for plugins):
+        (dir / "BUILD").write_text("python_sources(name='py'); pants_requirements(name='req')")
+        # We confirm the version from the main wheel, but only check that the testutil code can
+        # be imported at all.
+        (dir / "example.py").write_text(
+            "from pants import version, testutil; print(version.VERSION)"
+        )
+        result = subprocess.run(
+            ["pants", "run", "example.py"],
```

Comment:
This is currently failing for me locally with errors like:

```
ProcessExecutionFailure: Process 'Building py.pex with 2 requirements: pantsbuild.pants.testutil==2.19.0.dev2, pantsbuild.pants==2.19.0.dev2' failed with exit code 1.
stdout:

stderr:
pid 2365 -> /Users/huon/.cache/pants/named_caches/pex_root/venvs/ffd14ea6ad35e138cd78619cb1d7a53586ad1333/abee7c499dda8be23f517597ba8eeeff0a6f591a/bin/python -sE /Users/huon/.cache/pants/named_caches/pex_root/venvs/ffd14ea6ad35e138cd78619cb1d7a53586ad1333/abee7c499dda8be23f517597ba8eeeff0a6f591a/pex --disable-pip-version-check --no-python-version-warning --exists-action a --no-input --isolated -q --cache-dir /Users/huon/.cache/pants/named_caches/pex_root/pip_cache --log /private/var/folders/sv/vd266m4d4lvctgs2wpnhjs9w0000gn/T/pants-sandbox-faZ5NY/.tmp/pex-pip-log.150j3m9h/pip.log download --dest /Users/huon/.cache/pants/named_caches/pex_root/downloads/resolver_download.zyt05w7b/opt.homebrew.Cellar.python@3.9.3.9.16.Frameworks.Python.framework.Versions.3.9.bin.python3.9 pantsbuild.pants.testutil==2.19.0.dev2 pantsbuild.pants==2.19.0.dev2 --index-url https://pypi.org/simple/ --retries 5 --timeout 15 exited with 1 and STDERR:
ERROR: Could not find a version that satisfies the requirement pantsbuild.pants.testutil==2.19.0.dev2 (from versions: 1.23.0, 1.23.1rc0, 1.24.0rc0, 1.24.0rc1, 1.24.0rc2, 1.24.0, 1.24.1rc0, 1.24.1rc1, 1.25.0rc0, 1.25.0rc1, 1.25.0, 1.25.1rc1, 1.26.0rc0, 1.26.0rc1, 1.26.0rc2, 1.26.0, 1.26.1rc0, 1.27.0rc0, 1.27.0rc1, 1.27.0rc2, 1.27.0, 1.27.1rc0, 1.28.0rc0, 1.28.0rc1, 1.28.0rc2, 1.28.0, 1.28.1rc0, 1.29.0rc0, 1.29.0rc1, 1.29.0rc2, 1.29.0rc3, 1.29.0rc4, 1.29.0, 1.30.0rc0, 1.30.0rc1, 1.30.0rc2, 1.30.0, 1.30.1rc0, 1.30.1rc1, 1.30.1rc2, 1.30.1, 1.30.2rc0, 1.30.2rc1, 1.30.2rc2, 1.30.2rc3, 1.30.2, 1.30.3rc0, 1.30.3, 1.30.4rc0, 1.30.4, 1.30.5rc0, 1.30.5rc1, 2.0.0.dev0, 2.0.0.dev1, 2.0.0.dev2, 2.0.0.dev3, 2.0.0.dev4, 2.0.0.dev5, 2.0.0.dev6, 2.0.0.dev7, 2.0.0.dev8, 2.0.0.dev9, 2.0.0a0, 2.0.0a1, 2.0.0a2, 2.0.0a3, 2.0.0b0, 2.0.0b1, 2.0.0b2, 2.0.0b3, 2.0.0rc0, 2.0.0rc1, 2.0.0rc2, 2.0.0rc3, 2.0.0, 2.0.1rc0, 2.0.1rc1, 2.0.1rc2, 2.0.1rc3, 2.0.1rc4, 2.0.1, 2.0.2rc0, 2.0.2, 2.1.0.dev0, 2.1.0rc0, 2.1.0rc1, 2.1.0rc2, 2.1.0rc3, 2.1.0, 2.1.1rc0, 2.1.1rc1, 2.1.1rc2, 2.1.1, 2.1.2rc0, 2.1.2, 2.2.0.dev0, 2.2.0.dev1, 2.2.0.dev2, 2.2.0rc0, 2.2.0rc1, 2.2.0rc2, 2.2.0, 2.2.1rc0, 2.2.1rc1, 2.2.1rc2, 2.2.1, 2.2.2rc0, 2.2.2rc1, 2.2.2, 2.2.3rc0, 2.2.3, 2.2.4rc0, 2.2.4, 2.3.0.dev0, 2.3.0.dev1, 2.3.0.dev2, 2.3.0.dev3, 2.3.0rc0, 2.3.0rc1, 2.3.0rc2, 2.3.0rc3, 2.3.0rc4, 2.3.0, 2.3.1rc0, 2.3.1rc1, 2.3.1rc2, 2.3.1rc3, 2.3.1, 2.3.2rc0, 2.3.2, 2.3.3rc0, 2.3.3, 2.4.0.dev0, 2.4.0.dev1, 2.4.0.dev2, 2.4.0.dev3, 2.4.0rc0, 2.4.0rc1, 2.4.0rc2, 2.4.0rc3, 2.4.0rc4, 2.4.0rc5, 2.4.0, 2.4.1rc0, 2.4.1rc1, 2.4.1, 2.4.2rc0, 2.4.2rc1, 2.4.2, 2.5.0.dev0, 2.5.0.dev1, 2.5.0.dev2, 2.5.0.dev3, 2.5.0rc0, 2.5.0rc1, 2.5.0rc2, 2.5.0rc3, 2.5.0, 2.5.1rc0, 2.5.1rc1, 2.5.1rc2, 2.5.1rc3, 2.5.1rc4, 2.5.1rc5, 2.5.1rc6, 2.5.1, 2.5.2rc0, 2.5.2rc1, 2.5.2rc2, 2.5.2rc3, 2.5.2, 2.6.0.dev0, 2.6.0.dev1, 2.6.0.dev2, 2.6.0rc0, 2.6.0rc1, 2.6.0rc2, 2.6.0rc3, 2.6.0rc4, 2.6.0, 2.6.1rc0, 2.6.1rc1, 2.6.1rc2, 2.6.1rc3, 2.6.1, 2.7.0.dev0, 2.7.0.dev1, 2.7.0.dev2, 2.7.0.dev3, 2.7.0.dev4, 2.7.0rc0, 2.7.0rc1, 2.7.0rc2, 2.7.0rc3, 2.7.0rc4, 2.7.0rc5, 2.7.0, 2.7.1rc0, 2.7.1rc1, 2.7.1, 2.7.2rc0, 2.7.2rc1, 2.7.2rc2, 2.7.2, 2.8.0.dev0, 2.8.0.dev1, 2.8.0.dev2, 2.8.0.dev3, 2.8.0.dev4, 2.8.0.dev5, 2.8.0rc0, 2.8.0rc1, 2.8.0rc2, 2.8.0rc3, 2.8.0rc4, 2.8.0rc5, 2.8.0rc6, 2.8.0, 2.8.1rc0, 2.8.1rc1, 2.8.1rc2, 2.8.1, 2.9.0.dev0, 2.9.0.dev1, 2.9.0.dev2, 2.9.0.dev3, 2.9.0.dev4, 2.9.0rc0, 2.9.0rc1, 2.9.0rc2, 2.9.0rc3, 2.9.0rc4, 2.9.0rc5, 2.9.0rc6, 2.9.0, 2.9.1rc0, 2.9.1rc1, 2.9.1rc2, 2.9.1, 2.9.2rc0, 2.9.2, 2.10.0.dev0, 2.10.0.dev1, 2.10.0.dev2, 2.10.0.dev3, 2.10.0rc0, 2.10.0rc1, 2.10.0rc2, 2.10.0rc3, 2.10.0rc4, 2.10.0rc5, 2.10.0, 2.10.1rc0, 2.10.1rc1, 2.10.1, 2.11.0.dev0, 2.11.0.dev1, 2.11.0.dev2, 2.11.0.dev3, 2.11.0rc0, 2.11.0rc1, 2.11.0rc2, 2.11.0rc3, 2.11.0rc4, 2.11.0rc5, 2.11.0rc6, 2.11.0, 2.11.1rc0, 2.11.1rc1, 2.11.1rc2, 2.11.1rc3, 2.11.1, 2.12.0.dev0, 2.12.0.dev1, 2.12.0.dev2, 2.12.0.dev3, 2.12.0a0, 2.12.0rc0, 2.12.0rc1, 2.12.0rc2, 2.12.0rc3, 2.12.0, 2.12.1rc0, 2.12.1rc1, 2.12.1rc2, 2.12.1rc3, 2.12.1rc4, 2.12.1, 2.13.0.dev0, 2.13.0.dev1, 2.13.0.dev2, 2.13.0.dev3, 2.13.0.dev4, 2.13.0.dev5, 2.13.0a0, 2.13.0a1, 2.13.0rc0, 2.13.0rc1, 2.13.0rc2, 2.13.0rc3, 2.13.0, 2.13.1rc0, 2.13.1rc1, 2.13.1rc2, 2.13.1, 2.14.0.dev0, 2.14.0.dev1, 2.14.0.dev2, 2.14.0.dev3, 2.14.0.dev4, 2.14.0.dev5, 2.14.0.dev6, 2.14.0a0, 2.14.0a1, 2.14.0rc0, 2.14.0rc1, 2.14.0rc2, 2.14.0rc3, 2.14.0rc4, 2.14.0rc5, 2.14.0, 2.14.1rc0, 2.14.1, 2.14.2rc0, 2.14.2rc1, 2.14.2rc2, 2.14.2, 2.15.0.dev0, 2.15.0.dev1, 2.15.0.dev2, 2.15.0.dev3, 2.15.0.dev4, 2.15.0.dev5, 2.15.0a0, 2.15.0a1, 2.15.0rc0, 2.15.0rc1, 2.15.0rc2, 2.15.0rc3, 2.15.0rc4, 2.15.0rc5, 2.15.0rc6, 2.15.0rc7, 2.15.0, 2.15.1rc0, 2.15.1rc1, 2.15.1rc2, 2.15.1rc3, 2.15.1, 2.15.2rc0, 2.15.2, 2.16.0.dev0, 2.16.0.dev1, 2.16.0.dev2, 2.16.0.dev3, 2.16.0.dev4, 2.16.0.dev5, 2.16.0.dev6, 2.16.0.dev7, 2.16.0a0, 2.16.0a1, 2.16.0rc0, 2.16.0rc1, 2.16.0rc2, 2.16.0rc3, 2.16.0rc4, 2.16.0rc5, 2.16.0rc6, 2.16.0rc7, 2.16.0, 2.16.1rc0, 2.16.1rc3, 2.17.0.dev0, 2.17.0.dev1, 2.17.0.dev2, 2.17.0.dev3, 2.17.0.dev4, 2.17.0.dev5, 2.17.0a1, 2.17.0rc0, 2.17.0rc1, 2.17.0rc2, 2.17.0rc3, 2.17.0rc4, 2.17.0rc5, 2.17.0, 2.17.1rc0, 2.18.0.dev0, 2.18.0.dev1, 2.18.0.dev3, 2.18.0.dev4, 2.18.0.dev5, 2.18.0.dev6, 2.18.0.dev7, 2.18.0a0)
ERROR: No matching distribution found for pantsbuild.pants.testutil==2.19.0.dev2
```

Am I doing something wrong here? The packages certainly seem to be listed in https://wheels.pantsbuild.org/simple/ but that URL doesn't seem to be listed in the PEX invocation.